hack/deployer/runner/ocp.go

```diff
@@ -425,20 +446,19 @@ func (d *OcpDriver) bucketParams() map[string]interface{} {
 
 func (d *OcpDriver) runInstallerCommand(action string) error {
 	params := map[string]interface{}{
-		"ClusterStateDir":     d.runtimeState.ClusterStateDir,
+		"ClusterStateDirBase": filepath.Base(d.runtimeState.ClusterStateDir),
 		"HomeVolume":          SharedVolumeName(),
 		"GCloudCredsPath":     filepath.Join("/home", GCPDir, ServiceAccountFilename),
 		"OCPToolsDockerImage": d.runtimeState.ClientImage,
 		"Action":              action,
 	}
 	cmd := NewCommand(`docker run --rm \
-		-v {{.HomeVolume	}}:/home \
-		-v {{.ClusterStateDir}}:/config \
+		-v {{.HomeVolume}}:/home \
```

Comment:
I think this volume mount can be confusing as it is not the `/home` directory which is mounted here but the home directory of the current user.

Reply:
I tried to make it clearer by adding a comment and renaming the template parameter.